content/advanced_usage/pipelines_as_code.md

```diff
@@ -36,89 +36,42 @@ make it more declarative, depending on the plugin and possibly give more control
 Pipelines can currently be stored using JSON or YAML.
 
 
-Since GoCD 18.12.0 release, a new Config repositories page was added. This page lists existing config repos and allows CRUD (Create-Read-Update-Delete) operations for a config repo. This page also show errors and allows users to force a check of the repository.
+GoCD (18.12.0 release) now have a new Config repositories page. This page lists existing config repos and allows CRUD (Create-Read-Update-Delete) operations for a config repo. This page also show errors and allows users to force a check of the repository.
```

Comment:
Since these docs are versioned, why not just say: "The config repositories page (Admin -> Config Repositories) lists existing ..."?